install-and-configure/install/multi-cluster/federated-etl/thanos-migration-guide.md

```diff
@@ -0,0 +1,146 @@
+# Migration Guide from Thanos to Kubecost 2.0 (Aggregator)
+
+This tutorial is intended to help our users migrate from the legacy Thanos federation architecture to [Kubecost 2.0 (Aggregator)](aggregator.md). There are a few requirements in order to successfully migrate to Kubecost 2.0. This new version of Kubecost includes a new backend Aggregator which handles the ETL data built from source metrics more efficiently. Kubecost 2.0 provides new features, optimizes UI performance, and enhances the user experience.
+
+## Key changes
+
+* Assets and Allocations are now paginated using `offset`/`limit` parameters
+* New data available for querying every 2 hours (can be adjusted)
+* Substantial query speed improvements even when pagination not in effect
+* Data ingested into and queried from Aggregator component instead of directly from bingen files
+* Idle (sharing), Cluster Management sharing, and Network are computed a priori
```

Comment:
I think this is an unnecessary technical detail, let me know if you disagree

Reply:
Specifically the line about Idle? I agree! I think its worthwhile to keep the other lines though.

Reply:
Yes, just the idle line. Agreed on the rest

Reply:
Updated